AWS/SecretsManager/SecretRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ToucanTech\AWS\SecretsManager;
+
+use Aws\SecretsManager\Exception\SecretsManagerException;
+use Aws\SecretsManager\SecretsManagerClient;
+
+class SecretRepository {
+
+    /**
+     * Always pin the version of AWS APIs rather than using "latest" to avoid nasty shocks
+     */
+    public const SECRETSMANAGER_VERSION = '2017-10-17';
+
+    private $secretsManagerClient;
+
+    public function __construct(SecretsManagerClient $secretsManagerClient) {

```

Comment:
:bookmark: Suggestion: As we are using type hinting for the parameters I would recommend using type hints for the return values too. Just to help make more robust and help flag errors easier in the IDE for developers not using these methods correctly.